src/app/dashboard-widgets/create-work-item-widget/create-work-item-widget.component.spec.ts

```diff
@@ -4,10 +4,11 @@ import { By } from '@angular/platform-browser';
 import { ActivatedRoute, Router, RouterModule } from '@angular/router';
 import { NgArrayPipesModule } from 'angular-pipes';
 import { FilterService, WorkItem, WorkItemService } from 'fabric8-planner';
+import { NgLetModule } from 'fabric8-planner/app/shared/ng-let';
 import { Broadcaster } from 'ngx-base';
-import { Context, Contexts } from 'ngx-fabric8-wit';
+import { Context, Contexts, Space } from 'ngx-fabric8-wit';
```

Comment:
There are unused imports to clean up: `WorkItem`, `UserService`, `ConnectableObservable`, `publish`

Reply:
Done